library/src/main/java/net/sparkminds/library/entity/Session.java

```diff
@@ -0,0 +1,30 @@
+package net.sparkminds.library.entity;
+
+import java.time.LocalDateTime;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Builder;
+
+@Entity
+@Table(name = "session")
+@Builder
+public class Session {
```

Comment:
Em tạo thêm 1 cái entity common chứa những thông tin auditing
refer: https://mayankposts.medium.com/database-auditing-in-spring-boot-with-spring-security-context-and-spring-data-jpa-9215b43744bb